### PR TITLE
[SampleSQLBuilder]LPS-125421

### PR DESCRIPTION
**Pull request by slnn**

### CODE REVIEW

**Comment by slnn** [comment]
@zhangyan0701  Could you help me scan the test results?

Thanks
Lily

**Comment by zhangyan0701** [comment]
No regressions found.

File: modules/util/portal-tools-sample-sql-builder/src/main/java/com/liferay/portal/tools/sample/sql/builder/SampleSQLBuilder.java [hunk 1]
```diff
@@ -318,28 +351,80 @@ protected void mergeSQL(File inputFile, FileChannel outputFileChannel)
...
+			String line;
+
+			while ((line = reader.readLine()) != null) {
+				writer.append(line);
+				writer.append(System.lineSeparator());
+			}
+		}
+	}
+
+	private void _loadCreateSQLsFromClassLoader(Writer writer)
```
Comment (tinatian) [inline modules/util/portal-tools-sample-sql-builder/src/main/java/com/liferay/portal/tools/sample/sql/builder/SampleSQLBuilder.java:387]: @slnn Please rename this to **_loadCreateSQLs**, the method name is not consistent with the parameter, when you say load from class loader, usually you would have a class loader parameter, and here you actually do not need a class loader parameter as you always use the same one.

Tina

**Comment by tinatian** [comment]
@slnn Please see my comment and update this pull, other looks good to me.

Thanks.
Tina

**Comment by tinatian** [comment]
@slnn  Please see my comment and update this pull.

Tina

**Comment by slnn** [comment]
Hi @tinatian

Sorry for the response delayed, have updated according to the comment

Lily

**Comment by tinatian** [comment]
@shuyangzhou This pull is good to go.

Tina

**Comment by shuyangzhou** [comment]
See https://github.com/brianchandotcom/liferay-portal/pull/121570

**Pull request closed without merging**